src/style.module.css

```diff
@@ -146,10 +149,6 @@
   fill: var(--rdp-accent-color);
 }
 
-.root[dir="rtl"] .nav .chevron {
-  transform: rotate(180deg);
```

Comment:
These are duplicated styles that I removed, you can see the same thing applied in the next lines.

Reply:
Good find! Thanks!